var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -0,0 +1,38 @@
+from spack import *
+
+
+class Magma(Package):
```

Comment:
Can you update this to use `CMakePackage`? It handles most of the default CMake flags automatically.